overlord/devicestate/devicemgr.go

```diff
@@ -927,9 +928,19 @@ func (m *DeviceManager) ensureInstalled() error {
 
 	m.ensureInstalledRan = true
 
-	tasks := []*state.Task{}
+	var prev *state.Task
 	setupRunSystem := m.state.NewTask("setup-run-system", i18n.G("Setup system for run mode"))
-	tasks = append(tasks, setupRunSystem)
+
+	tasks := []*state.Task{setupRunSystem}
+	addTask := func(t *state.Task) {
+		t.WaitFor(prev)
+		tasks = append(tasks, t)
+		prev = t
+	}
+	prev = setupRunSystem
+
+	makeBootable := m.state.NewTask("restart-system-to-run-mode", i18n.G("Ensure next boot to run mode"))
```

Comment:
maybe s/makeBootable/restartSystem/ or something like that?

Reply:
ah yes, good point